### PR TITLE
try importing libvmi in VMIAddressSpace

### PR DESCRIPTION
**Pull request by Wenzel**
This should fix the error reported by @bridgeythegeek in #374 

I'm testing this now

### CODE REVIEW

**Comment by bridgeythegeek** [comment]
Thanks for jumping on this :)

But, quick sanity check... does line 5:
```
has_libvmi = True
```
over-write the good work done in the `try/except` immediately above?

**Comment by Wenzel** [comment]
That's why i was testing it =)
I'm writing the fix

**Comment by bridgeythegeek** [comment]
Works for me! :trophy: 

As for style, I'd say both are fine. The first is very clear, but the second is probably more "pythonic".

**Comment by Wenzel** [comment]
Thanks for your feedback.
The lesson we learned from this if that we need a travis CI with some basic tests on the Rekall repo :)

File: rekall-core/rekall/plugins/addrspaces/vmi.py [hunk 1]
```diff
@@ -1,4 +1,9 @@
-from libvmi import Libvmi
+try:
```
Comment (scudette) [inline rekall-core/rekall/plugins/addrspaces/vmi.py]: How about the simpler:
```
libvmi = None
try:
   from libvmi import libvmi
except ImportError: pass
```


Alternatively we can just conditionally import this entire file in the\_\_init\_\_.py file

**Comment by scudette** [comment]
On Monday, 5 March 2018, Mathieu Tarral <notifications@github.com> wrote:

> Thanks for your feedback.
> The lesson we learned from this if that we need a travis CI with some
> basic tests on the Rekall repo :)
>

We do have a large test suite but it is too slow to run on travis. We used
to run our own VM for this but it has been decomissioned. I will look into
reinstating that, or maybe run a very simple smoke test on travis.

Thanks
Michael.

>

**Comment by Wenzel** [comment]
can we merge this ?
are you waiting on other modifications ?

**Pull request merged**